modules/apps/commerce/commerce-account-service/src/main/java/com/liferay/commerce/account/internal/upgrade/v10_1_0/AccountEntryUpgradeProcess.java

```diff
@@ -0,0 +1,79 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.commerce.account.internal.upgrade.v10_1_0;
+
+import com.liferay.commerce.payment.model.CommercePaymentMethodGroupRel;
+import com.liferay.commerce.product.constants.CommerceChannelAccountEntryRelConstants;
+import com.liferay.commerce.product.service.CommerceChannelAccountEntryRelLocalService;
+import com.liferay.petra.string.StringBundler;
+import com.liferay.portal.kernel.upgrade.UpgradeProcess;
+
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+
+/**
+ * @author Crescenzo Rega
+ */
+public class AccountEntryUpgradeProcess extends UpgradeProcess {
+
+	public AccountEntryUpgradeProcess(
+		CommerceChannelAccountEntryRelLocalService
+			commerceChannelAccountEntryRelLocalService) {
+
+		_commerceChannelAccountEntryRelLocalService =
+			commerceChannelAccountEntryRelLocalService;
+	}
+
+	@Override
+	protected void doUpgrade() throws Exception {
+		_updateDefaultPaymentMethods();
```

Comment:
you are not executing multiple sub-methods, so you could put everything inside here.

Reply:
I have verified and there are several classes

- AssetDisplayPrivateLayoutUpgradeProcess
- AssetDisplayLayoutFriendlyURLPrivateLayoutUpgradeProcess
...
- LayoutTemplateIdUpgradeProcess
- ...
- WikiPageResourceUpgradeProcess
- WikiNodeUpgradeProcess
- ...
- MonitoringConfigurationUpgradeProcess
...
- KaleoProcessUpgradeProcess
...
- UpgradeLayoutTypeSettings

 that use a private method within the public doUpgrade () method, for this reason I used the same approach

Reply:
most of them have multiple private sub-methods, here we are not making things any cleaner